testing/testcafe/tests/editors/list/focus.ts

```diff
@@ -0,0 +1,87 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { testScreenshot } from '../../../helpers/themeUtils';
+import url from '../../../helpers/getPageUrl';
+import List from '../../../model/list';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`List`
+  .page(url(__dirname, '../../container.html'));
+
+const createList = (selectionMode) => createWidget('dxList', {
+  items: ['item1', 'item2', 'item3'],
+  showSelectionControls: true,
+  selectionMode,
+});
+
+test('Should apply styles on selectAll checkbox after tab button click', async (t) => {
```

Comment:
let's change "click" to "press" everywhere, it's a little confusing
```suggestion
test('Should apply styles on selectAll checkbox after tab button press', async (t) => {
```

Reply:
fixed